avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -677,12 +678,16 @@ private static Object maybeWrapSchemaless(Schema schema, Object value, String ty
   }
 
   public org.apache.avro.Schema fromConnectSchema(Schema schema) {
-    return fromConnectSchema(schema, new HashMap<Schema, org.apache.avro.Schema>());
+    return fromConnectSchema(schema,
+        new HashMap<>(),
+        new HashMap<>());
   }
 
-  public org.apache.avro.Schema fromConnectSchema(Schema schema,
-                                                  Map<Schema, org.apache.avro.Schema> schemaMap) {
-    return fromConnectSchema(schema, schemaMap, false);
+  public org.apache.avro.Schema fromConnectSchema(
+      Schema schema,
+      Map<Schema, org.apache.avro.Schema> schemaMap,
+      Map<String, org.apache.avro.Schema> fromConnectCycles) {
```

Comment:
Call me strange, but I don't think I understand what `fromConnectCycles` means. I understand `fromConnectData` or `fromConnectSchema`. But what are the cycles in connect? A better name would help understand faster what we store in those maps IMO.

Reply:
Having said that about variable name, let me say that we now definitely need javadocs here.